Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.9-slim
```

Comment:
When I was training the model I was getting [this error](https://github.com/tensorflow/tensorflow/issues/60673). To use an older version of tensorflow I had to use an older version of Python, and I didn't want to use two different version of Python in the project so I downgraded the Python version everywhere. I can probably upgrade it again now though because the TensorFlow on M1 issue has been resolved.